src/box/errcode.h

```diff
@@ -431,6 +431,7 @@ struct errcode_record {
 	 */ \
```

Comment:
We decided, that from now on we specify since which version feature is present in DocBot request. It'll be `3.3.0`, i suppose? Doc team struggles a lot to figure this out.

Reply:
Tbh, I don't know which version is the next one. Why not 3.2.0? I am a bit out of touch on the versions topic, sorry.

Reply:
We wanted to release `3.2.0` recently, but it still has not been released and there's new `3.1.1`, so it looks like `3.2.0` is the next one indeed

Reply:
Ok, I mentioned 3.2.0 in the docbot request.